civis/io/_tables.py

```diff
@@ -33,6 +34,36 @@
 }
 
 
+SQL_PANDAS_MAP = {'smallint': np.int,
+               'int': np.int,
+               'integer': np.int,
+               'bigint': np.int,
+               'decimal': np.float,
+               'numeric': np.float,
+               'float': np.float,
+               'float4': np.float,
+               'float8': np.float,
+               'real': np.float,
+               'double precision': np.float,
+               'boolean': np.bool,
+               'bool': np.bool,
+               'int2': np.int,
+               'int4': np.int,
+               'int8': np.int,
+               'char': np.str,
+               'character': np.str,
+               'nchar': np.str,
+               'bpchar': np.str,
+               'varchar': np.str,
+               'character varying': np.str,
+               'nvarchar': np.str,
+               'text': np.str}
+
+# Checkout timestamptz
```

Comment:
Can you make this comment more verbose?

Reply:
I can just delete this comment because I got all of the aliases